README.md

```diff
@@ -46,6 +47,8 @@ Tired of typing lengthy git commands? Well, if you are amongst who feel like me,
 | `git stash drop [<stash>]`| `gsh-d [<stash>]`| Yes | [:information_source:](https://git-scm.com/docs/git-stash#git-stash-apply--index-q--quietltstashgt) |
 | `git tag -l --sort=v:refname \| tail -n8` | `gt` | No | [:information_source:](https://git-scm.com/docs/git-tag) |
 | `git add . && git commit -m` | `g.c <message>` | No, it accepts only commit message |[:information_source:](https://git-scm.com/docs/git-commit) |
+| `git rebase` | `grb <branch name>` | No| [:information_source:](https://git-scm.com/docs/git-rebase) |
+| `git remote set-url origin` | `gremote <repourl>` | No| [:information_source:](https://git-scm.com/docs/git-remote) |
```

Comment:
Can you remove `gremote` as there is an alias already for it and it accepts all the default parameters with it.